app/experimenter/experiments/tests/factories/nimbus.py

```diff
@@ -110,9 +110,7 @@ class Lifecycles(Enum):
     LIVE_ENROLLING = LAUNCH_APPROVE_APPROVE + (LifecycleStates.LIVE_IDLE_ENROLLING,)
     LIVE_ENROLLING_WAITING = LIVE_ENROLLING + (LifecycleStates.LIVE_WAITING_ENROLLING,)
     LIVE_PAUSED = LIVE_ENROLLING + (LifecycleStates.LIVE_IDLE_PAUSED,)
-    ENDING_REVIEW_REQUESTED = LAUNCH_APPROVE_APPROVE + (
-        LifecycleStates.LIVE_REVIEW_ENDING,
-    )
+    ENDING_REVIEW_REQUESTED = LIVE_PAUSED + (LifecycleStates.LIVE_REVIEW_ENDING,)
```

Comment:
@jaredlockhart I'd like to double check this is right with you. It seems like when an experiment is ending,`sEnrollmentPaused` would be set to `true`.

Reply:
It may or may not, you can end an experiment before you pause it.

Reply:
But this is fine either way.